### PR TITLE
add meta tags to render links on social media

### PR DESCRIPTION
**Pull request by teonbrooks**

### CODE REVIEW

**Comment by teonbrooks** [comment]
@wlach, I think these are the meta tags needed to make the links rendered nicely on twitter and facebook. since we don't have small descriptions, I just duped the title for the description. since I haven't taken the time to properly learn django, I'm not certain if that is the correct pattern for inserted the title into the html

File: server/templates/base.html [hunk 1]
```diff
@@ -8,6 +8,17 @@
     <link rel="icon" type="image/png" sizes="16x16" href="{% static 'assets/favicon-16x16.png' %}">
     <link rel="icon" type="image/x-icon" href="{% static 'assets/favicon.ico' %}">
     <link rel="mask-icon" href="{% static 'assets/safari-pinned-tab.svg' %}" color="#5bbad5">
+    <!-- Add these meta properties to render domain on Facebook -->
+		<meta property="og:title" content="{{ title }}">
+		<meta property="og:description" content="{{ title }}">
+		<meta property="og:image" content="https://files.gitter.im/iodide-project/iodide/xW1J/iodide-sticker-2.png">
+		<meta property="og:url" content="https://iodide.io">
```
Comment (wlach) [inline server/templates/base.html]: This should be derived from `settings.SITE_URL`

File: server/templates/base.html [hunk 2]
```diff
@@ -8,6 +8,17 @@
     <link rel="icon" type="image/png" sizes="16x16" href="{% static 'assets/favicon-16x16.png' %}">
     <link rel="icon" type="image/x-icon" href="{% static 'assets/favicon.ico' %}">
     <link rel="mask-icon" href="{% static 'assets/safari-pinned-tab.svg' %}" color="#5bbad5">
+    <!-- Add these meta properties to render domain on Facebook -->
+		<meta property="og:title" content="{{ title }}">
+		<meta property="og:description" content="{{ title }}">
+		<meta property="og:image" content="https://files.gitter.im/iodide-project/iodide/xW1J/iodide-sticker-2.png">
```
Comment (wlach) [inline server/templates/base.html]: This should be just `{% static 'assets/apple-touch-icon.png' %}` as above, shouldn't it?

File: server/templates/base.html [hunk 3]
```diff
@@ -8,6 +8,17 @@
...
+		<meta property="og:url" content="https://iodide.io">
+		<!-- Add these meta properties are for twitter -->
+		<meta name="twitter:title" content="{{ title }}">
+		<meta name="twitter:description" content="{{ title }}">
+		<!-- <meta name="twitter:site" content="@iodide"> -->
+		<meta name="twitter:image" content="https://files.gitter.im/iodide-project/iodide/xW1J/iodide-sticker-2.png">
```
Comment (wlach) [inline server/templates/base.html]: Same as for image above

**Comment by wlach** [comment]
hey @teonbrooks, will you have time to look into this in the next couple days? I'd like to get this in before the next iodide release

**Comment by wlach** [comment]
I adjusted the comments for the meta tags, but other than that, this looks great! I will merge.

**Pull request merged**